### ISSUE TITLE
English revisions: Table header cells missing text

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#85]
URL: https://hcdev.openplus.ca/en/node/1109/revisions (click on Build for any Recall or Alert from Dashboard and then Revisions tab)
User Agent: Chrome, Firefox
Language: Both

Issue Description: Revisions tab (in Build) has a table that is missing text in the header cells - `<th>` is empty. Table headers `<th>` should not be empty.

Steps to reproduce: On Dashboard, open a Recall/Alert by clicking on Build. Go to Revisions tab and review code and use WAVE toolbar.

Expected results: `<th>` tag in table should not be empty. If it cannot have any content then the `<th>` tag should be changed to `<td>`.

![image](https://user-images.githubusercontent.com/14280062/129375667-e52e55dc-2745-4d5c-9cf6-d0147e367807.png)

![image](https://user-images.githubusercontent.com/14280062/129375985-224231ce-d2e9-4265-a0af-1da12e01d377.png)

**Comment by qanitah** [comment]
Fixed
